### DISCUSSION TITLE
Can actor-init-sparql-rdfjs work in the browser?

### DISCUSSION THREAD

**Discussion by linonetwo** [comunica/comunica#443] [Q&A] [answered]
#### Issue type:

- :question: Question

____
#### Question:

https://github.com/comunica/comunica/blob/bc62dd66ffcb90082157239c722cdf051ebea0f0/packages/actor-init-sparql-rdfjs/index.ts#L12

This uses `__dirname` and `/config/config-default.json`, seems very node.js, can this run on the browser?

**Comment by rubensworks** [comment, marked as answer]
Yes, actor-init-sparql-rdfjs can definitely run in the browser.
However, the index file you reference won't be used when you webpack/browserify comunica, instead this file will be used: https://github.com/comunica/comunica/blob/bc62dd66ffcb90082157239c722cdf051ebea0f0/packages/actor-init-sparql-rdfjs/index-browser.ts
(That is because of the [`"browser"` entry in our `package.json` file`](https://github.com/comunica/comunica/blob/bc62dd66ffcb90082157239c722cdf051ebea0f0/packages/actor-init-sparql-rdfjs/package.json#L99-L101).

**Comment by linonetwo** [comment]
Thank you! I got it. And it actually works.
